apps/test/util/testUtils.js

```diff
@@ -121,17 +122,7 @@ exports.runOnStudioTick = function (tick, fn) {
  * Generic function allowing us to hook into onTick. Only tested for Studio/Applab
  */
 exports.runOnAppTick = function (app, tick, fn) {
-  if (!app) {
-    throw new Error('not supported outside of studio/applab');
-  }
-  var ran = false;
-  app.onTick = _.wrap(app.onTick, function (originalOnTick) {
-    if (app.tickCount === tick && !ran) {
-      ran = true;
-      fn();
-    }
-    originalOnTick();
-  });
+  tickWrapper.runOnAppTick(app, tick, fn);
```

Comment:
It's a little silly to have these live in testUtils and in tickWrapper, but going through and cleaning up all callers seems like a pain. If you think it's worth it, I can make an effort to do that.